package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "dependencies": {
     "chalk": "~0.4.0",
-    "lodash": "^3.10.1",
+    "lodash": "^4.5.0",
```

Comment:
I update it in order to use lodash.lowerFirst() in script-base.js, because that function is only available at 4.5.0, but I could use another method to lowercase the first char :)